op-challenger/fault/abi_test.go

```diff
@@ -26,7 +27,10 @@ func setupFaultDisputeGame() (common.Address, *bind.TransactOpts, *backends.Simu
 	if err != nil {
 		return common.Address{}, nil, nil, nil, err
 	}
-	backend := backends.NewSimulatedBackend(core.GenesisAlloc{from: {Balance: big.NewInt(params.Ether)}}, 50_000_000)
+	backend := backends.NewSimulatedBackend(
```

Comment:
Why a diff here?

Reply:
line-length formatter - if it's important i can revert back

Reply:
What length is it set to? Running your own linter that's relatively different than the style of the codebase causes a huge amount of churn.